src/main/webapp/WEB-INF/component/activity-helper.jsp

```diff
@@ -62,22 +68,23 @@ public String formatCreationTime(Instant time) {
 // Reusable method that returns the layout for the user.
```

Comment:
Since this is a public method, consider adding javadoc: http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html